doc/release-notes/0.36/0.36.md

```diff
@@ -32,6 +32,8 @@ OpenJ9 release 0.36.0 supports OpenJDK 8, 11, 17.
 
 All releases are tested against the OpenJ9 functional verification (FV) test suite, the OpenJDK test suites, and additional tests provided by Adoptium.
 
+Support for running OpenJDK 8 and OpenJDK 11 on CentOS 6.10 is deprecated and might be removed in a future release.
```

Comment:
Should this be under Notable changes? I'm concerned it may be missed here.

Reply:
I have moved the text to the table. Is this OK?